src/bluehawk/actions/copy.ts

```diff
@@ -100,6 +120,11 @@ export const copy = async (args: CopyArgs): Promise<string[]> => {
     );
     const targetPath = path.join(directory, document.basename);
 
+    const copyInfo: CopyInfo = {
```

Comment:
```suggestion
    const copyInfo = {
```

nit: `CopyResult.filesCopied` is already a `CopyInfo[]` so it should be type safe to just push this later on. The explicit type doesn't hurt for now though so feel free to leave it in.

Reply:
Usually I prefer less explicit typing, but in this case the explicit type ensures the instantiation (this line to ~127) matches the type. If we remove the typing and the type changes, then the error will appear when you push rather than instantiate. Not a huge deal, but that might be a head scratcher for someone.